src/newshop.cpp

```diff
@@ -110,19 +112,19 @@ bool is_ok_char(struct char_data * keeper, struct char_data * ch, vnum_t shop_nr
     return TRUE;
   }
 
-  if ((shop_table[shop_nr].races.IsSet(RACE_HUMAN) && GET_RACE(ch) == RACE_HUMAN) ||
+  if ((shop_table[shop_nr].races.IsSet(RACE_HUMAN) && (GET_RACE(ch) == RACE_HUMAN || GET_RACE(ch) == RACE_DRAKE_HUMAN)) ||
```

Comment:
Should ghouls be in here too? Is it possible to have a yes-ghoul-but-no-ghoul-trolls vendor?

Reply:
Good call, re-added ghouls and tested yes-ghoul-but-no-ghoul-troll mix-and-match, should be working fine. Ghouls are now properly discriminated against by both their condition and their metatype, just in case they forgot that no one loves them.